broker/src/main/fabric8/deployment.yml

```diff
@@ -8,10 +8,10 @@ spec:
           periodSeconds: 5
           timeoutSeconds: 1
           tcpSocket:
-            port: 5672
+            port: 5671
```

Comment:
I was wondering if we could use the HTTP based Jolokia interface instead (or liveness and health checks) that is started up with Artemis?